Makefile

```diff
@@ -118,65 +118,17 @@ $(BIN_DIR)/azure-acr-credential-provider.exe: $(PKG_CONFIG) $(wildcard cmd/acr-c
 ##@ Images
 ## --------------------------------------
 
-.PHONY: docker-pull-prerequisites
-docker-pull-prerequisites: ## Pull prerequisite images.
-	docker pull docker/dockerfile:1.3.1
-	docker pull docker.io/library/golang:1.17-buster
-	docker pull gcr.io/distroless/static:latest
-
 buildx-setup:
 	docker buildx inspect img-builder > /dev/null || docker buildx create --name img-builder --use
 	# enable qemu for arm64 build
 	# https://github.com/docker/buildx/issues/464#issuecomment-741507760
 	docker run --privileged --rm tonistiigi/binfmt --uninstall qemu-aarch64
 	docker run --rm --privileged tonistiigi/binfmt --install all
 
-.PHONY: build-ccm-image
-build-ccm-image: buildx-setup docker-pull-prerequisites ## Build controller-manager image.
-	docker buildx build \
-		--pull \
-		--output=type=$(OUTPUT_TYPE) \
-		--platform linux/$(ARCH) \
-		--build-arg ENABLE_GIT_COMMAND="$(ENABLE_GIT_COMMAND)" \
-		--build-arg ARCH="$(ARCH)" \
-		--build-arg VERSION="$(VERSION)" \
-		--file Dockerfile \
-		--tag $(IMAGE) .
-
-.PHONY: build-node-image-linux
-build-node-image-linux: buildx-setup docker-pull-prerequisites ## Build node-manager image.
```

Comment:
I personally like this target because when for most the case, people want to build amd64, they can just use `build-node-image-linux`, instead of `build-node-image-linux-amd64`, and that's one fo the reason arch is default to `amd64`

Reply:
OUTPUT_TYPE="docker,dest=$mydir/azure-cloud-node-manager.tar"  is supported by design. Added two target to build ccm-amd64 and cnm-linux.